docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -46,15 +46,33 @@ memory availability. Supports the following values:
 * ``total-reservation-on-blocked-nodes`` - Kill the query currently using the
   most memory specifically on nodes that are now out of memory.
 
+``query.low-memory-task-killer.policy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** ``total-reservation-on-blocked-nodes``
+
+Configures the behavior to handle killing running tasks in the event of low
+memory availability. Supports the following values:
```

Comment:
From context it seems like this property is **only** used when `retry-policy` is set to `TASK`. If that's the case, should add a sentence to this description saying that, and whether the `query` low memory killer does anything in that case.

Reply:
What we should be sayint to users is:
`query.low-memory-task-killer.policy` is used when`retry_policy` is `TASK`.
`query.low-memory-query-killer.policy` is used otherwise.

In reality it is that:
`query.low-memory-task-killer.policy` is used for queries which were run with `retry_policy` set to tasks and `query.low-memory-query-killer.policy` for other queries - but we officially do not support mixed workloads so we should not word it like that.

Reply:
O - actualy we use such wording already:

```
    Does not apply for queries with task level retries enabled (``retry-policy=TASK``)
```

I will keep it consistent for now then